src/Frame.js

```diff
@@ -0,0 +1,112 @@
+var Frame = function (game) {
```

Comment:
Really love that you've extracted the frame logic out from the game.  Is there a way to make it not know about the game? Have a look at the exemplar for how to tell frames about the other frames they need to know about.